src/views/components/define-signatory-info/signatory-details.njk

```diff
@@ -1,11 +1,26 @@
 {% from "govuk/components/radios/macro.njk" import govukRadios %}
 {% from "govuk/components/input/macro.njk"  import govukInput %}
 
+{% set signatoryId = (signatory.id | lower) %}
+{% set fieldsetId = "signatory_" + signatoryId %}
+
+{% set isSigningName = "isSigning_" + signatoryId %}
+{% set isSigningId = "is-signing_" + signatoryId %}
+
+{% set directorEmailName = "directorEmail_" + signatoryId %}
+{% set directorEmailId = "director-email_" + signatoryId %}
+
+{% set onBehalfNameName = "onBehalfName_" + signatoryId %}
+{% set onBehalfNameId = "on-behalf-name_" + signatoryId %}
+
+{% set onBehalfEmailName = "onBehalfEmail_" + signatoryId %}
+{% set onBehalfEmailId = "on-behalf-email_" + signatoryId %}
```

Comment:
I felt this was cleaner to define the form element names at the top rather than having string concatenation dotted throughout the file.